### PR TITLE
Publish 22.0.0.9-beta

### PR DESCRIPTION
**Pull request by mbroz2**
https://blogs-staging-openlibertyio.ng.bluemix.net/blog/2022/08/04/jakarta-core-profile-22009-beta.html
image

### CODE REVIEW

File: posts/2022-08-04-jakarta-core-profile-22009-beta.adoc [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+
+To use MicroProfile OpenAPI 3.1, add the `mpOpenAPI-3.1` feature to your server.xml:
+[source,xml]
+----
+<featureManager>
+    <feature>mpOpenAPI-3.1</feature>
+</featureManager>
+----
+
+Next, deploy your rest application and then visit the `/openapi` endpoint to view the OpenAPI documentation, or visit `/openapi/ui` for a user interface where you can browse through it. You can enhance the documentation by using annotations to add things like textual descriptions to each method. The annotations are available as a maven dependency:
```
Comment (dmuelle) [inline posts/2022-08-04-jakarta-core-profile-22009-beta.adoc:276]:
```suggestion
Next, deploy your rest application and then visit the `/openapi` endpoint to view the OpenAPI documentation, or visit `/openapi/ui` for a user interface where you can browse through it. You can enhance the documentation by using annotations to add things like textual descriptions to each method. The annotations are available as a Maven dependency:
```

File: posts/2022-08-04-jakarta-core-profile-22009-beta.adoc [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+
+
+[#cdi]
+== CDI 4.0
+CDI allows objects to be bound to lifecycle contexts, to be injected, to be associated with interceptors and decorators, and to interact in a loosely coupled fashion by firing and observing events.
+Highlights of CDI 4.0 include support for build compatible extensions and observable container state events. This update also brings in new Jakarta EE 10 versions of the Jakarta Annotations and Jakarta Interceptors APIs.
+
+=== Build compatible extensions
+
+Previous versions of CDI allowed users to provide Portable Extensions to customize the CDI application initialization lifecycle. These Build Compatible Extensions make implementing extensions amenable to build-time processing. To implement a Build Compatible Extension, provide an implementation of the `BuildCompatibleExtension` interface, declared via `META-INF/services`. The implementation can provide methods annotated with one of the extension annotations, each of which corresponds to the extension execution phases;
```
Comment (dmuelle) [inline posts/2022-08-04-jakarta-core-profile-22009-beta.adoc:128]:
```suggestion
Previous versions of CDI allowed users to provide Portable Extensions to customize the CDI application initialization lifecycle. These Build Compatible Extensions make implementing extensions amenable to build-time processing. To implement a Build Compatible Extension, provide an implementation of the `BuildCompatibleExtension` interface, declared via `META-INF/services`. The implementation can provide methods annotated with one of the extension annotations, each of which corresponds to the extension execution phases:
```

File: posts/2022-08-04-jakarta-core-profile-22009-beta.adoc [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+
+The Liberty `<cdi12>` `server.xml` configuration element has been superseded by a new `<cdi>` element, which applies to CDI versions 1.2 and later. For example:
+
+[source, xml]
+----
+<cdi enableImplicitBeanArchives="false" emptyBeansXmlCDI3Compatibility="true"/>
+----
+
+* The `enableImplicitBeanArchives` attribute is the same as it was in previous versions.
+  - If set to `true`, which is the default, then archives that do not contain a `beans.xml` file are treated as Implicit Bean Archives and scanned for classes that have Bean Defining Annotations.
```
Comment (dmuelle) [inline posts/2022-08-04-jakarta-core-profile-22009-beta.adoc:220]:
```suggestion
  - If set to `true`, which is the default, then archives that do not contain a `beans.xml` file are treated as Implicit Bean Archives and scanned for classes that have bean-defining annotations.
```

**Review by dmuelle — APPROVED** [review]
couple minor typos- approving so as not to block

**Comment by mbroz2** [comment]
Made the changes locally, will update along with concurrency 3.0

**Pull request merged**